packages/channels/botpress-channel-web/src/views/web/messages/form.jsx

```diff
@@ -103,20 +107,32 @@ export default class Form extends Component {
     if (!this.props.elements) {
       return null
     }
-    const elements = this.props.elements.map(fe => <FormElement parent={this} {...this.props} {...fe} />)
+    const elements = this.props.elements.map((fe, index) => (
+      <FormElement key={`${index}-form-element`} parent={this} {...this.props} {...fe} />
+    ))
     return (
-      <div className={style.formOverlay}>
-        <form className={style.formContainer} onSubmit={this.handleSubmit.bind(this)}>
-          <div onClick={this.handleClose.bind(this)} className={style.formClose}>
+      <div className={`${style.formOverlay} botpress__message-element-form-overlay`}>
+        <form
+          className={`${style.formContainer} botpress__message-element-form-container`}
+          onSubmit={this.handleSubmit.bind(this)}
+        >
+          <div
+            onClick={this.handleClose.bind(this)}
+            className={`${style.formClose} botpress__message-element-form-close`}
+          >
             <svg version="1.1" width="15" height="15" xmlns="http://www.w3.org/2000/svg">
-              <line x1="1" y1="15" x2="15" y2="1" stroke="grey" stroke-width="2" />
-              <line x1="1" y1="1" x2="15" y2="15" stroke="grey" stroke-width="2" />
+              <line x1="1" y1="15" x2="15" y2="1" stroke="grey" strokeWidth="2" />
+              <line x1="1" y1="1" x2="15" y2="15" stroke="grey" strokeWidth="2" />
             </svg>
           </div>
-          <div className={style.formTitle}>{this.props.title}</div>
+          <div className={`${style.formTitle} botpress__message-element-form-title`}>{this.props.title}</div>
           {elements}
-          <div className={style.buttonLayer}>
-            <input className={style.formSubmit} type="submit" value="Submit" />
+          <div className={`${style.buttonLayer} botpress__message-element-form-button-layer`}>
+            <input
+              className={`${style.formSubmit} botpress__message-element-form-submit`}
```

Comment:
Also not the best name here, should it be `botpress__message-element-form-submit-button`?